server/models/visualization.js

```diff
@@ -15,6 +15,11 @@ const dataSchema = new mongoose.model("Visualization",
             required: true,
             unique: true,
             type: String
+        },
+        csvLink: {
+            required: true,
```

Comment:
Small, non-blocking, lets have this as not required, not sure if all visualizations will need one.